pinot-core/src/main/java/org/apache/pinot/core/auth/Actions.java

```diff
@@ -0,0 +1,153 @@
+    public static final String DELETE_TIME_BOUNDARY = "DeleteTimeBoundary";
+    // Used in /tables/{tableName} API with state as one of the parameters
+    public static final String DISABLE_TABLE = "DisableTable";
+    public static final String DOWNLOAD_SEGMENT = "DownloadSegment";
+    // Used in /tables/{tableName} API with state as one of the parameters
+    public static final String DROP_TABLE = "DropTable";
+    // Used in /tables/{tableName} API with state as one of the parameters
+    public static final String ENABLE_TABLE = "EnableTable";
+    public static final String FORCE_COMMIT = "ForceCommit";
+    public static final String GET_BROKER = "GetBroker";
+    public static final String GET_CONFIG = "GetConfig";
+    public static final String GET_CONSUMING_SEGMENTS = "GetConsumingSegments"; // SK:
+    public static final String GET_CONTROLLER_JOBS = "GetControllerJobs"; // SK:
+    public static final String GET_DEBUG_INFO = "GetDebugInfo";
+    public static final String GET_EXTERNAL_VIEW = "GetExternalView";
+    public static final String GET_IDEAL_STATE = "GetIdealState";
+    public static final String GET_INSTANCE = "GetInstance";
+    public static final String GET_METADATA = "GetMetadata";
+    public static final String GET_PARTITION = "GetPartition";
+    public static final String GET_PAUSE_STATUS = "GetPauseStatus";
+    public static final String GET_ROUTING_TABLE = "GetRoutingTable";
+    public static final String GET_SCHEMA = "GetSchema";
+    public static final String GET_SEGMENT = "GetSegment";
+    public static final String GET_SEGMENT_LINEAGE = "GetSegmentLineage";
+    public static final String GET_SEGMENT_MAP = "GetSegmentMap";
+    public static final String GET_SERVER_MAP = "GetServerMap";
+    public static final String GET_SIZE = "GetSize";
+    public static final String GET_STATE = "GetState";
+    public static final String GET_STORAGE_TIER = "GetStorageTier";
+    public static final String GET_TABLE_CONFIG = "GetTableConfig";
+    public static final String GET_TABLE_LEADER = "GetTableLeader";
+    public static final String GET_TIME_BOUNDARY = "GetTimeBoundary";
+    public static final String PAUSE_CONSUMPTION = "PauseConsumption";
+    public static final String QUERY_TABLE = "QueryTable";
+    public static final String REBALANCE_TABLE = "RebalanceTable";
+    public static final String REBUILD_BROKER_RESOURCE = "RebuildBrokerResource";
+    public static final String REFRESH_ROUTING = "RefreshRouting";
+    public static final String RELOAD_SEGMENT = "ReloadSegment";
+    public static final String REPLACE_INSTANCE = "ReplaceInstance";
+    public static final String REPLACE_SEGMENT = "ReplaceSegment";
+    public static final String RESUME_CONSUMPTION = "ResumeConsumption";
+    public static final String UPDATE_CONFIG = "UpdateConfig";
+    public static final String UPDATE_SCHEMA = "UpdateSchema";
+    public static final String UPDATE_TABLE = "UpdateTable";
```

Comment:
When managing a table, we have 3 scenarios:
1. TableConfig
2. Schema
3. TableConfigs (TableConfig & schema)

We want to name them more explicitly, e.g. `<verb><TableConfig|Schema|TableConfigs>`